### PR TITLE
:page_with_curl: Set build number automatically with script

### PR DESCRIPTION
**Pull request by Odrakir**
Gets the build number from the number of commits and does this for every target.

### CODE REVIEW

File: teferiScripts/bump.rb [hunk 1]
```diff
@@ -51,17 +41,15 @@ def bumpMajor
...
 
 verb = ARGV[0]
-validCommands = [ "build", "hotfix", "minor", "major"]
+validCommands = ["hotfix", "minor", "major"]
 
 commandDescriptions = {
-	"build" => "increments build number",
 	"hotfix" => "increments maintenance version and build number",
 	"minor" => "increments minor version and build number",
 	"major" => "increments major version and build number"
```
Comment (amulware) [inline teferiScripts/bump.rb]: Fix descriptions please (and maybe include in minor and major that lower numbers are reset to 0 as well.

Comment (Odrakir) [inline teferiScripts/bump.rb]: This doesn't change anything when you run the app in Xcode (simulator or device). It just sets the build number in the executable or when archiving, but nothing changes in code of config files, so nothing to commit.

I'll change the descriptions, but I don't understand what you mean by "and maybe include in minor and major that lower numbers are reset to 0 as well."

Comment (amulware) [inline teferiScripts/bump.rb]: Unless the script was changed, when you increase the minor, it will reset the hotfix number. I.e.: 0.1.1 -> 0.2.0

**Comment by sofokli1** [comment]
@Odrakir shouldn't this close #549 ?

**Comment by Odrakir** [comment]
Correct

**Pull request merged**